tensorflow/python/keras/applications/resnet.py

```diff
@@ -544,7 +544,7 @@ def decode_predictions(preds, top=5):
       and width and height should be no smaller than 32.
       E.g. `(200, 200, 3)` would be one valid value.
     pooling: Optional pooling mode for feature extraction
-      when `include_top` is `False`.
+      when `include_top` is `False`. It could be:
       - `None` means that the output of the model will be
           the 4D tensor output of the
           last convolutional block.
```

Comment:
I would rewrite all of these to "If `None` then the output of the model...`

Reply:
That's a good idea! I was a bit hesitant to tamper with the existing documentation as I'm adding one. 
I'll rewrite it in no time.

Reply:
Added a new commit changing the pooling argument description.